cppseminar/presentation/Pages/Submissions/Detail.cshtml.cs

```diff
@@ -60,6 +60,12 @@ public async Task OnGetAsync([Required]string id)
                     if ((await _authService.AuthorizeAsync(
                         User, oneCase, AuthorizationConstants.Submit)).Succeeded)
                     {
+                        try
+                        {
+                            var testCount = await _testService.GetTestCountAsync(User.GetEmail(), oneCase.Id);
+                            oneCase.SubmittedCount = testCount.Count;
+                        }
+                        catch (Exception ex) { }
```

Comment:
It is ok. Logging is done deeper, inside of GetTestCountAsync, as that function has more context.

This is done just to prevent crash. If SubmittedCount is not set, html will show something like error : unknown... 
I think it is better that way, because not being able to tell remaining test count, is not very important and user can continue with his work on webpage

Reply:
In recent push I have added comment in there, so I believe this is resolved